### ISSUE TITLE
Display messages when converting gtfs to gps in function gtfs2gps

### ISSUE THREAD

**Issue by kauebraga** [ipeaGIT/gtfs2gps#60]
As it may take a long time to complete the conversion between gtfs to gps, it would be nice to have some messages informing the process that is happening at the moment.

**Comment by rafapereirabr** [comment]
Perhaps it would be important to show messages / warnings alerting to cases of `shape_ids` and `trip_ids` that were not properly processed. I'm not sure how that would work since `shape_ids` are processed in parallel, what kind of messes up with printing messages along the way.

In any case, I've added 3 messages along the function. They are not super helpful but it's better than nothing. Moreover, the code also shows a progress bar to alleviate user's code anxiety :) 

```
message("converting shapes and stops to sf objects")

message("Unzipping and reading GTFS.zip file")

message("Processing the data")
```

**Comment by rafapereirabr** [comment]
Hi all. I've checked the code closely today to see where we could add more messages as suggested by Kaue. However, most of the data procesing steps are run in parallel, so printing additional messages would not work anyway. I'm afraid we will only have the three messages indicated above but I hope the progress bar will help alleviate users' code anxiety . Closing this issue for now
